### ISSUE TITLE
Skeleton for verification page of metadata (1/3)

### ISSUE THREAD

**Issue by TangoYankee** [openoakland/woeip#88]
**Is your feature request related to a problem? Please describe.**
A page where users can view, verify, and edit the meta data that was automatically generated for the session.

**Describe the solution you'd like**
The user should be brought to a page where they are presented with a form, pre-filled with data. The user can change the values of the data in the fields. The user can either save the data or cancel the upload. Saving progresses to the session view page. Canceling returns the user to the upload page.

Login is required for this view

**Describe alternatives you've considered**
Manually entering all fields, when initially uploading data.

**Additional context**
Data is coming from an S3 bucket and a staging table. Only the meta-data is extracted and verified. After user confirmation of the meta-data, all of the air quality and its metadata are saved to the SQL database.

**Labels**
- [x] Add a status label
  - `backlog`
  - `discussing`
  - `ready`
  - `implementing`
- [x] Add additional labels as needed

### For Developers
**Affected Components**
This issue is one of three dealing with the verification page. The three issues will create a skeleton of the necessary components, fill the skeleton with data from the S3 bucket and Staging Table or handle errors, and finally add styling & accessibility to the page.

This issue outlines the components needed for the page:
- Form
- Form test
- View (Excluding integration of staged data)
- View test (Excluding integration of staged data)
- url
- HTML template, with minimal tagging
- redirects to other web-pages 

**Technical Resources**
- [Django Forms](https://docs.djangoproject.com/en/2.2/topics/forms/)
- [Function-based views](https://docs.djangoproject.com/en/2.2/topics/http/views/)
  - (In contrast to class-based views)
- [Login Required Decorator](https://docs.djangoproject.com/en/2.2/topics/auth/default/#the-login-required-decorator)

**Design Documentation**
- [URL design card](https://trello.com/c/pzgbHYU7/82-url-mapping)
- [Wireframes](https://github.com/openoakland/woeip/files/2959738/WOEIP-UploadFlow-v3.pdf)

**Testing Resources**
[Django Testing](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Testing)

**Comment by theecrit** [comment]
Blocked by upload implementation.

**Comment by theecrit** [comment]
Closing as "won't fix" due to changes in tech stack. Now covered by issue #132.
